mysite/scripts/snapshot_then_push.sh

```diff
@@ -30,16 +30,37 @@ cd "$PREFIX"
 
 ## Create a path to store the JSON in
 TEMPFILE="$(mktemp --suffix -snapshot.json.gz)"
+TEMPDB = "$(mktemp --suffix -snapshot.db)"
 
 ## Do the slooow thing of snapshotting the database
 ./manage.py snapshot_public_data | gzip > "$TEMPFILE"
 
+## Create a new settings to specify the name of the temporary snapshot db
+touch mysite/snapshot_settings.py
```

Comment:
If you're creating the file with `>` below, no need to `touch` first. Not a show-stopper.